commerce/domain/shipping/cart/calculator/classes/ShippingCartCalculatorSample.cls

```diff
@@ -81,13 +81,23 @@ public class ShippingCartCalculatorSample extends CartExtension.ShippingCartCalc
             CartExtension.CartDeliveryGroupMethod cartDeliveryGroupMethod01 = new CartExtension.CartDeliveryGroupMethod('Ground Shipping', 10.99, shippingProduct);
             cartDeliveryGroupMethod01.setCarrier('USPS');
             cartDeliveryGroupMethod01.setClassOfService('Ground Shipping');
+            cartDeliveryGroupMethod01.setTransitTimeMin(1);
+            cartDeliveryGroupMethod01.setTransitTimeMax(3);
+            cartDeliveryGroupMethod01.setTransitTimeUnit(CartExtension.TimeUnitEnum.DAYS);
```

Comment:
@alakhmani Can you please include the enum values for CartExtension as it will not be obvious (not public) what these values are until the public doc is updated?